examples/workflows/dask.py

```diff
@@ -0,0 +1,44 @@
+"""This example showcases how to run Dask within a Hera submitted Argo workflow."""
+from hera.workflows import (
+    Steps,
+    Workflow,
+    script,
+)
+
+
+@script(image="ghcr.io/dask/dask:latest")
+def dask_computation(namespace: str = "default", n_workers: int = 1) -> None:
+    import subprocess
+
+    # this is required for otherwise the dask distributed and kubernetes clients packages are not included by default
+    # ideally, you'd have a package in your organization that takes care of the following cluster details :)
+    subprocess.run(
```

Comment:
This does not look great. Would it be possible to do something like what you propose here?
https://github.com/argoproj-labs/hera/discussions/665#discussioncomment-6220596